.github/workflows/continuous-integration.yml

```diff
@@ -30,4 +30,32 @@ jobs:
       - name: ${{ matrix.name }}
         uses: laminas/laminas-continuous-integration-action@v1
         with:
-          job: ${{ matrix.job }}
\ No newline at end of file
+          job: ${{ matrix.job }}
+
+  qa-alpine:
```

Comment:
DId you try to define this job same like `qa-ubuntu` with additional image with alpine?

Reply:
I have looked into `laminas-continous-integration-action`, but it doesn't seem to support Alpine Linux. There is a folder `setup/ubuntu/` which includes a Ubuntu-specific `setup.sh` script: https://github.com/laminas/laminas-continuous-integration-action/blob/1.15.x/setup/ubuntu/setup.sh. There seems to be no equivalent for Alpine Linux.